services/provider/server/server.go

```diff
@@ -874,13 +874,38 @@ func (s *OCSProviderServer) GetStorageClientsInfo(ctx context.Context, req *pb.S
 			continue
 		}
 
-		clientInfo := &pb.ClientInfo{ClientID: req.ClientIDs[i]}
+		clientInfo := &pb.ClientInfo{ClientID: req.ClientIDs[i], ClientProfiles: map[string]string{}}
 
 		consumerConfig := util.WrapStorageConsumerResourceMap(consumerConfigMap.Data)
 		if consumerConfig.GetRbdRadosNamespaceName() != "" {
 			clientInfo.RadosNamespace = consumerConfig.GetRbdRadosNamespaceName()
+			var fsid string
+			if cephCluster, err := util.GetCephClusterInNamespace(ctx, s.client, s.namespace); err != nil {
+				klog.Errorf("failed to get cephCluster in namespace %s: %v", s.namespace, err)
+				response.Errors = append(response.Errors,
+					&pb.StorageClientInfoError{
+						ClientID: req.ClientIDs[i],
+						Code:     pb.ErrorCode_Internal,
+						Message:  "failed loading client information",
+					},
```

Comment:
Is there a reason to add individual error for each client? If we cannot get an FSID then we will get an error for all clients